gateway/src/apicast/policy/keycloak_scope_check/keycloak_scope_check.lua

```diff
@@ -0,0 +1,157 @@
+--- Keycloak Scope Check Policy
+-- This policy exposes the realm roles and the client roles in the JWT.
+--
+--- The realm roles are specified when you want to add scope check to the every client's resources or 3scale itself.
+-- https://www.keycloak.org/docs/latest/server_admin/index.html#realm-roles
```

Comment:
Would be good to pin this to a particular version, so the link does not break over time.

Reply:
Changed.